src/main.rs

```diff
@@ -1,3 +1,88 @@
+use git2::{Commit, Error, Time};
+use git_graph::graph::GitGraph;
+
+struct Args {}
+
 fn main() {
-    println!("Hello, world!");
+    let args = Args {};
+    match run(&args) {
```

Comment:
I think having `main` return `Result<(), Error>` could serve the same purpose.

Reply:
It does, thanks!